winsock-demo/winsock-server/main.cpp

```diff
@@ -0,0 +1,10 @@
+#include ".\Server\Server.h"
+
+int main(void) {
+    Server server;
+    server.startServer();
```

Comment:
startServer means that we already started and ready to handle msg, so move handleMessages() into startServer()